src/js/structs/ServicesList.js

```diff
@@ -24,7 +22,7 @@ module.exports = class ServicesList extends List {
 
     if (filters) {
       if (filters.ids) {
-        services = _.filter(services, function (service) {
+        services = services.filter(function (service) {
           return this.ids.indexOf(service.id) !== -1;
         }, {ids: filters.ids});
```

Comment:
i think we might need to make some more changes here. here we pass `{ids: filters.ids}` as the 3rd argument to make it the context. the native `filter` doesn't have that